lib/testing/src/lib/process-services-cloud/components/filter.ts

```diff
@@ -0,0 +1,60 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { element, by } from 'protractor';
+import { BrowserVisibility } from '../../core/utils/browser-visibility';
+import { BrowserActions } from '../../core/utils/browser-actions';
+export class FilterComponent {
```

Comment:
I think this refers to edit-process-filter-cloud.component and edit-task-filter-cloud.component.md, which are two different components and are already covered in testing package by EditTaskFilterCloudComponentPage and EditProcessFilterCloudComponentPage POs, so probably this class is not needed.

Reply:
First of all, I don't see the reason of having another page object for the same component:
**process-filters-cloud-component.page.ts** and **edit-process-filter-cloud-component.page.ts**, respectively: **task-filters-cloud-component.page.ts** and **edit-task-filter-cloud-component.page.ts**. 
Most of the content of these files are duplication, the question is why do we have 2 filter components, when there is a single one (duplicated code is a living proof for that)?